pkg/swift/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "gluesql_swift"
```

Comment:
let's use dash instead of underscore, both are allowed but we prefer `-` in our projects.
`gluesql-swift`